terraform/modules/aws/waf_v2_acl/rules.tpl

```diff
@@ -0,0 +1,74 @@
+${jsonencode([
```

Comment:
Not sure if refining the rules is part of this but looks sensible, any idea how these rules compare with our existing or is that a future piece of work?

Reply:
I've just chucked some basic rules in for now. They will need to be refined.